tests/system_tests_http2.py

```diff
@@ -415,15 +415,20 @@ def test_000_stats(self):
         stats = qd_manager.query('io.skupper.router.httpRequestInfo')
         self.assertEqual(len(stats), 2)
 
-        # Give time for the core thread to augment the stats.
-        i = 0
-        while i < 3:
-            if not stats or stats[0].get('requests') < 2:
-                i += 1
-                sleep(1)
-                stats = qd_manager.query('io.skupper.router.httpRequestInfo')
-            else:
-                break
+        def check_num_requests():
+            statistics = qd_manager.query('io.skupper.router.httpRequestInfo')
+            for stat in statistics:
+                self.assertEqual(stat.get('requests'), 2)
+
+        # This test intermittently fail s with the following error -
+        # self.assertEqual(s.get('requests'), 2) - AssertionError: 1 != 2
+        # The logs do not show any request failures.
+        # Since this test is failing intermittently and not always, I am going to give more
+        # time for this test to execute. If this test still fails, I will look at the http2 code
+        # in more detail.
+        retry_exception(check_num_requests)
```

Comment:
This would retry on _any_ exception whatsoever, including `SyntaxError`, no? We don't have better function for retry now, but for the future I propose
* https://github.com/skupperproject/skupper-router/pull/439